src/main/java/org/terasology/fluid/system/FluidCommonSystem.java

```diff
@@ -45,6 +44,6 @@ public void preBegin() {
         ResourceUrn waterTextureUri = TextureUtil.getTextureUriForColor(Color.BLUE);
         Texture texture = assetManager.getAsset(waterTextureUri, Texture.class).get();
 
-        fluidRegistry.registerFluid("Fluid:Water", new TextureFluidRenderer(texture, "water"), LiquidType.WATER);
+        fluidRegistry.registerFluid("Fluid:Water", new TextureFluidRenderer(texture, "water"));
```

Comment:
the registerFluid method has been/is being changed for accommodating this change?
I am trying to do similar edit in [substanceMatters](https://github.com/e-aakash/SubstanceMatters/blob/master/src/main/java/org/terasology/substanceMatters/systems/FluidSubstanceCommonSystem.java#L53) module, but it is showing an error due to parameter mismatch

Reply:
Yup, so once this PR has been merged, the error will go away.
It's not been merged yet hence why the method will give you an error for now. That's fine though